test/runtests.jl

```diff
@@ -133,125 +136,144 @@ test_shapes = Dict(
     SubPolygon => SubPolygon([LinearRing{Point}(view(points, 1:4))]),
 )
 
-@testset "GeoInterface compatibility" begin
-    foreach(test_shapes) do (k, s)
-        @test GeoInterface.testgeometry(s)
-        @test GeoInterface.extent(s) isa GeoInterface.Extent
-        # This test should be moved to GeoInterface `testgeometry`
-        @test GeoInterface.ncoord(s) ==
-            (2 + GeoInterface.is3d(s) + GeoInterface.ismeasured(s)) ==
-            length(GeoInterface.coordnames(s))
-        # TODO Plotting a single measured point fails, needs fixing in GeoInterfaceRecipes.jl
-        GeoInterface.ismeasured(s) && GeoInterface.trait(s) isa GeoInterface.PointTrait || plot(s)
-    end
-    @test_broken GeoInterface.testgeometry(MultiPatch(Rect(1, 3, 2, 4), [0], [1], points, Interval(1, 4), [1, 2, 3, 4]))
+@testset "Shapefile.jl" begin
```

Comment:
Can we just untab all of this? 

There's no need to change anything except the added tests, there can be multiple test sets. 

We dont want all the git blame for the tests pointing at this PR.

Reply:
I untabbed the tests, the diff seems a lot more limited now.

Reply:
Thanks